tests/cases/conformance/types/contextualTypes/jsxAttributes/contextuallyTypedStringLiteralsInJsxAttributes01.tsx

```diff
@@ -2,9 +2,12 @@
 // @declaration: true
 
 namespace JSX {
-    interface IntrinsicElements {
+    export interface IntrinsicElements {

```

Comment:
Why does this change to `export` and why add `export interface Element`? Does this somehow make them undefined for this test?